### DISCUSSION TITLE
Only dark and light themes

### DISCUSSION THREAD

**Discussion by kud** [easonwong-de/Adaptive-Tab-Bar-Colour#93] [Q&A]
Hi there,

I hope you're doing well.

I wanted to suggest that the extension limit its theme options to just two when the user requests it: light and dark. This would allow users to choose their preferred theme based on the website they are visiting.

Personally, I usually prefer the light theme, but I switch to dark for sites like GitHub and Pocket Casts, which works well for me.

However, I've noticed that some websites incorporate yellow elements, particularly in their banners, which causes Firefox to adopt a yellow hue as well. This is something I really dislike. I would much prefer to have only light and dark themes available.

Is there a way to use this with the current options, or can we look into adding these specific options?

Thank you for considering my request.

**Comment by easonwong-de** [comment]
Hi, sorry for the late reply.

The behaviour you mentioned might be outside the scope of ATBC, as its core purpose is to make the browser interface adapt to web content.

**Reply by kud** [reply]
I feel you! Thank you for your answer. 😌
